src/lib/logical_query_plan/aggregate_node.cpp

```diff
@@ -57,16 +57,16 @@ std::string AggregateNode::description(const DescriptionMode mode) const {
   return stream.str();
 }
 
-const std::vector<std::shared_ptr<AbstractExpression>>& AggregateNode::column_expressions() const {
+std::vector<std::shared_ptr<AbstractExpression>> AggregateNode::column_expressions() const {
   // We do not return node_expressions directly here, because we do not want to expose ANY() to the following LQP
   // nodes. This way, we execute ANY() as intended, but do not have to traverse the LQP upwards and adapt nodes
   // that reference the ANY'd column.
-  _column_expressions.resize(node_expressions.size());
-  std::copy(node_expressions.begin(), node_expressions.end(), _column_expressions.begin());
+  auto column_expressions = std::vector<std::shared_ptr<AbstractExpression>>(node_expressions.size());
+  std::copy(node_expressions.begin(), node_expressions.end(), column_expressions.begin());
```

Comment:
Probably missing something, but how is this different from `auto column_expressions = node_expressions`?

Reply:
I think you are right. Same result, much simpler. 👍